Sources/VCReorderableStackView.swift

```diff
@@ -122,29 +122,51 @@ private extension ReorderableStackView {
     }
 
     func reorderViews(with gesture: UILongPressGestureRecognizer) {
-
+        
+        var snapshotViewFrame: CGFloat? = .zero
```

Comment:
We can be more precise with this name.
How about `snapshotViewMidPoint`?

Reply:
snapshotViewMidPoint very Nice!